firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -668,4 +675,13 @@ public void execute(@NonNull Runnable command) {
       HANDLER.post(command);
     }
   }
+
+  @Nullable
+  private static String detectKotlinVersion() {
+    try {
+      return kotlin.KotlinVersion.CURRENT.toString();
+    } catch (NoClassDefFoundError ex) {
+      return null;
```

Comment:
Can we write this as "no_kotlin_version" or "-1". Could give us a sense of what portion of clients do not have kotlin. I realise this can be derived already. But, seems more convenient

Reply:
I erred on the side of not populating what's not present, e.g. it's consistent with what is done for unity, flutter, cpp, etc. So I think I prefer to do it this way, but @davidair could correct me if that's not the case and/or your suggestion adds convenience.

Reply:
We normally only put things into the platform info that "are there" - and they vary between platform. It doesn't make sense to put the Xcode version into the Android string, so I would recommend omitting Kotlin if not present.